src/coreclr/jit/clrjit.natvis

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+Licensed to the .NET Foundation under one or more agreements.
+The .NET Foundation licenses this file to you under the MIT license.
+-->
+
+
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+
+  <Type Name="Compiler">
+    <DisplayString>{{ {info.compFullName,sb} ({info.compMethodHashPrivate,x})}}</DisplayString>
+    <StringView>info.compFullName,sb</StringView>
+  </Type>
+
+  <Type Name="BasicBlock">
+    <DisplayString Condition="bbJumpKind==BBJ_COND || bbJumpKind==BBJ_ALWAYS || bbJumpKind==BBJ_LEAVE || bbJumpKind==BBJ_EHCATCHRET || bbJumpKind==BBJ_CALLFINALLY">BB{bbNum,d}->BB{bbJumpDest->bbNum,d}; {bbJumpKind,en}</DisplayString>
+    <DisplayString>BB{bbNum,d};--- {bbJumpKind}</DisplayString>
+  </Type>
+
+  <Type Name="EHblkDsc">
+    <DisplayString>type={ebdHandlerType}</DisplayString>
+  </Type>
+
+  <!-- GenTree -->
+  <Type Name="GenTree">
+    <DisplayString>{{{gtOper,en}, {gtType,en}}}}</DisplayString>
```

Comment:
```suggestion
    <DisplayString>{{{gtOper,en}, {gtType,en}}}</DisplayString>
```

Reply:
actually, instead of `{{ }}`, I was thinking of wrapping them in `[` and `]` . That way, inside `natvis` file, we don't have to escape the `{`. Thoughts?

Reply:
I guess there's a question about whether the surrounding characters are needed at all; I would argue no.

Reply:
I couldn't figure out a way to not display the original address. With that being present, it is hard to differentiate the actual text of interest. E.g. refposition. 

![image](https://user-images.githubusercontent.com/12488060/118051460-ea459780-b335-11eb-9dcd-92e853061481.png)
vs.
![image](https://user-images.githubusercontent.com/12488060/118051505-ffbac180-b335-11eb-8d40-d3ff35e3d0e0.png)

Reply:
Sure, if you think it provides additional, useful visual differentiation from the leading address, then keeping the `[` or `{` is fine with me. I suggested removing it for the function name, for instance, because I don't think it helps. Although if you always use `[`, and that isn't otherwise (?) used by the debugger, then it might provide a visual indication that natvis has kicked in for this case.

Reply:
yes, I will keep the `[` around then.